<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWindow</class>
 <widget class="QMainWindow" name="SetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>290</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>210</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройка</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="BotSetupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="title">
       <string>Настройка ботов</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="BotNumberLabel">
         <property name="text">
          <string>Количество ботов :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="BotNumberSpinBox">
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="MazeSetupBox">
      <property name="title">
       <string>Настройка лабиринта</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="MazeHeightLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Вертикаль : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="MazeHeightSpinBox">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="specialValueText">
            <string/>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="value">
            <number>25</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="MazeWidthLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Горизоталь :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="MazeWidthSpinBox">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="value">
            <number>25</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="GenerateBtn">
         <property name="text">
          <string>Сгенерировать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="LoadFromFileBtn">
         <property name="text">
          <string>Загрузить из файла</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
